### PR TITLE
[zh-cn]: update the translation of Navigator: language property

### PR DESCRIPTION
**Pull request by T34-active**

### CODE REVIEW

File: files/zh-cn/web/api/navigator/language/index.md [hunk 1]
```diff
@@ -1,41 +1,42 @@
...
-**`Navigator.language`** 为只读属性，返回一个表示用户偏好语言（通常是浏览器界面语言）的字符串。
+只读属性 **`Navigator.language`** 返回一个字符串，表示用户的首选语言，通常是浏览器 UI 的语言。
 
-## 语法
+## 值
 
-```js
-const lang = navigator.language;
-```
+一个字符串，表示语言版本，如 {{RFC(5646, "语言识别标签（也称为 BCP 47）")}} 中定义。有效的语言代码示例包括“en”、“en-US”、“fr”、“fr-FR”、“es-ES”等。
```
Comment (yin1999) [inline files/zh-cn/web/api/navigator/language/index.md:14]:
```suggestion
一个表示语言版本的字符串，定义于 {{RFC(5646, "语言识别标签（也称为 BCP 47）")}}中。有效的语言代码示例包括“en”、“en-US”、“fr”、“fr-FR”、“es-ES”，等。
```

File: files/zh-cn/web/api/navigator/language/index.md [hunk 2]
```diff
@@ -1,41 +1,42 @@
...
 
-### 值
+请注意，在 iOS 10.2 之前的 Safari 浏览器中，返回的国家代码是小写的：例如“en-us”、“fr-fr”。
```
Comment (yin1999) [inline files/zh-cn/web/api/navigator/language/index.md:16]:
```suggestion
请注意，在 iOS 10.2 之前的 Safari 浏览器中，返回的国家代码是小写的：例如“en-us”、“fr-fr”，等。
```

File: files/zh-cn/web/api/navigator/language/index.md [hunk 3]
```diff
@@ -1,41 +1,42 @@
...
 
-一个 {{domxref("DOMString")}}。`lang`存储了一个在 [BCP 47](https://tools.ietf.org/rfc/bcp/bcp47.txt) 中定义的语言版本。一些有效的语言版本代码示例包括： “en”、“en-US”、“fr”、“fr-FR”、“es-ES”等。
+## 示例
 
-请注意，在 iOS 10.2 之前的 Safari 中，返回的语言版本代码是小写的： “en-us”、“fr-fr”等。
+### 使用 Intl 构造函数进行语言特定格式化
```
Comment (yin1999) [inline files/zh-cn/web/api/navigator/language/index.md:20]:
```suggestion
### 使用 Intl 构造函数进行特定于语言的格式化
```

**Review by yin1999 — APPROVED** [review]
Thanks @T34-active

File: files/zh-cn/web/api/navigator/language/index.md [hunk 4]
```diff
@@ -1,41 +1,42 @@
...
-**`Navigator.language`** 为只读属性，返回一个表示用户偏好语言（通常是浏览器界面语言）的字符串。
+只读属性 **`Navigator.language`** 返回一个字符串，表示用户的首选语言，通常是浏览器 UI 的语言。
 
-## 语法
+## 值
 
-```js
-const lang = navigator.language;
-```
+一个表示语言版本的字符串，定义于 {{RFC(5646, "语言识别标签（也称为 BCP 47）")}}中。有效的语言代码示例包括“en”、“en-US”、“fr”、“fr-FR”、“es-ES”，等。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/navigator/language/index.md:14]:
```suggestion
一个表示语言版本的字符串，定义于 {{RFC(5646, "语言识别标签（也称为 BCP 47）")}}中。有效的语言代码示例包括“en”、“en-US”、“fr”、“fr-FR”、“es-ES”等。
```

File: files/zh-cn/web/api/navigator/language/index.md [hunk 5]
```diff
@@ -1,41 +1,42 @@
...
 
-### 值
+请注意，在 iOS 10.2 之前的 Safari 浏览器中，返回的国家代码是小写的：例如“en-us”、“fr-fr”，等。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/navigator/language/index.md:16]:
```suggestion
请注意，在 iOS 10.2 之前的 Safari 浏览器中，返回的国家代码是小写的：例如“en-us”、“fr-fr”等。
```

File: files/zh-cn/web/api/navigator/language/index.md (hunk 4 above)
Comment (yin1999) [reply files/zh-cn/web/api/navigator/language/index.md]: 本身是可加逗号的，可能是我平时的习惯如此，去掉也没问题

https://wap.peopleapp.com/article/rmh22206407/rmh22206407

Comment (T34-active) [reply files/zh-cn/web/api/navigator/language/index.md]: > 本身是可加逗号的，可能是我平时的习惯如此，去掉也没问题
> 
> https://wap.peopleapp.com/article/rmh22206407/rmh22206407

无伤大雅

**Pull request merged**